dev/io.openliberty.microprofile6.internal_fat/bnd.bnd

```diff
@@ -0,0 +1,31 @@
+#*******************************************************************************
+# Copyright (c) 2019, 2021 IBM Corporation and others.
```

Comment:
I assume the files in this project have been mostly copied unchanged which is why the copyright dates are in the past?

I'm assuming this file has changes to update the java level to 11 so I think it's date should have changed. Were there any other files in this project which also needed changes?

Reply:
Yes, it was copied, yes the date should be updated.